src/lens.js

```diff
@@ -25,7 +25,9 @@ var map = require('./map');
  *      R.over(xLens, R.negate, {x: 1, y: 2});  //=> {x: -1, y: 2}
  */
 module.exports = _curry2(function lens(getter, setter) {
-  return _curry2(function(f, s) {
-    return map(function(v) { return setter(v, s); }, f(getter(s)));
-  });
+  return function(f) {
+    return function(s) {
+      return map(function(v) { return setter(v, s); }, f(getter(s)));
+    };
+  };
 });
```

Comment:
For reasons I don't yet understand it's necessary for a lens to be a unary function in order for lenses to compose.